mkdocs/api/model-selection.md

```diff
@@ -0,0 +1,16 @@
+# Model Selection
+
+:::sklego.model_selection.GroupTimeSeriesSplit
+    options:
+        show_root_full_path: false
```

Comment:
![CleanShot 2023-10-30 at 17 05 04](https://github.com/koaning/scikit-lego/assets/1019791/d61e00ac-2d62-4235-b81a-a19f2bf5ebb4)

params also don't render super well on this one

Reply:
- For the table, the best solution I found is to make it in code as
![image](https://github.com/koaning/scikit-lego/assets/42817048/b6250468-e445-447f-951b-67e5e70cdcbb)
  Since the table concept is slightly different in mkdocs.
- Thanks for the heads up on the parameters, the ":" at the end has been wrongly copy-pasted in a few places